.autocopr/podman.spec

```diff
@@ -51,7 +51,11 @@ BuildRequires: ostree-devel
 BuildRequires: systemd
 BuildRequires: systemd-devel
 Requires: conmon >= 2:2.0.30-2
-Requires: containers-common >= 4:1-30
+%if 0%{?fedora}
+Requires: containers-common >= 4:1-21
```

Comment:
Why the downgrade?

Reply:
@edsantiago f34 is on `4:1-21`, I figure a simple change should be ok since it's not affecting mainstream podman users.

Reply:
but if you prefer separation between f34 , f35 etc. I could do that.

Reply:
see podman IRC, f34 only has 1.21

Reply:
Ack, thanks. This is fine, I just didn't have the context.

Reply:
ack, i noticed your comment just now, but made the finer changes already. PTAL.

Reply:
@edsantiago na, I reverted to the earlier simpler change :) . PTAL now.